src/rt/cpp/cown.h

```diff
@@ -195,33 +206,32 @@ namespace verona::cpp
   private:
     /// Underlying cown that has been acquired.
     /// Runtime is actually holding this reference count.
-    cown_ptr<T> origin_cown;
+    ActualCown<T>* origin_cown;

```

Comment:
Can this not be a reference?  We don't ever have a null pointer here do we?

Reply:
Correct